docs/action-type-template.asciidoc

```diff
@@ -22,7 +22,9 @@ or as needed when you're creating a rule.
 <ACTION-TYPE> connectors have the following configuration properties:
 
 ////
-List of user-facing connector configurations. This should align with the fields available in the Create connector flyout form for this connector type.
+List of user-facing connector configurations.
+This should align with the fields available in the Create connector flyout form for this connector type.
+To include these configuration details in the API documentation, add apprpriate files in x-pack/plugins/actions/docs/openapi/components/schemas/ and reference them from oas_docs/overlays/connectors.overlays.yaml
```

Comment:
I think appropriate is misspelled here

Reply:
Ha!  You are right!

Reply:
Fixed in e49f23741b54533b11048ad5907283e8e78639af